integrations/fastembed/src/haystack_integrations/components/embedders/fastembed/fastembed_text_SPLADE_embedder.py

```diff
@@ -0,0 +1,125 @@
+from typing import Any, Dict, List, Optional, Union
+import numpy as np
+from haystack import component, default_to_dict
+
+from .embedding_backend.fastembed_backend import _FastembedSparseEmbeddingBackendFactory
+
+
+@component
+class FastembedTextSPLADEEmbedder:
```

Comment:
`FastembedTextSPLADEEmbedder` should be `FastembedTextSparseEmbedder`?
Since it won't be limited to SPLADE models only But will have other sparse models as well.

Reply:
Complete Renaming done :)